service/src/main/java/greencity/security/service/LoginAttemptServiceImpl.java

```diff
@@ -33,20 +36,15 @@ public Integer load(final String key) {
     /**
      * {@inheritDoc}
      */
+    @Override
     public void loginFailed(final String key) {
-        int attempts;
-        try {
-            attempts = attemptsCache.get(key);
-        } catch (final ExecutionException e) {
-            attempts = 0;
-        }
-        attempts++;
-        attemptsCache.put(key, attempts);
+        attemptsCache.asMap().merge(key, 1, Integer::sum);
```

Comment:
how do we remove it after `blockTimeInHours` ends?

Reply:
good remark, now I will think about how to implement it